setup.cfg

```diff
@@ -78,6 +78,7 @@ tests =
   pytest-mock
   pytest-mpl>=0.12 # First version to support our figure tests
   tox
+  tox-pypi-filter
```

Comment:
Why is this needed?

Reply:
I added this since we don't actually pre-install it. Tox never uses the test pin files.

Reply:
We could copy astropy and use to to control the env in which tox runs: https://github.com/astropy/astropy/blob/master/tox.ini#L10

This does expose us to some edge cases, but in general should work fine.

Reply:
What kind of edge cases?

Reply:
I'll make this change in another PR.